.travis.yml

```diff
@@ -61,18 +48,16 @@ before_deploy:
   - source /home/travis/google-cloud-sdk/path.bash.inc
   - gcloud --quiet version
   - gcloud --quiet components update
-  - gcloud --quiet components beta update
   - gcloud --quiet components update kubectl
+  - curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash
```

Comment:
If we want the latest from today, we should point the the commit hash of today, but we should likely not allow accidental version bumps, as it will just break, only question is how long it will take :)